build/cli.cjs

```diff
@@ -1538,6 +1538,212 @@ async function importResponse(oldPtauFilename, contributionFilename, newPTauFile
 
 }
 
+/*
+    Copyright 2018 0KIMS association.
+    This file is part of snarkJS.
+    snarkJS is a free software: you can redistribute it and/or modify it
+    under the terms of the GNU General Public License as published by
```

Comment:
We should discuss with someone from legal what the consequences of using software which has a GPL-3 license. Iiuc, it implies that all derived software should also have this license, but it might just affect this modified version of snarkjs.